core/src/main/java/org/openedx/core/system/notifier/PushEvent.kt

```diff
@@ -0,0 +1,7 @@
+package org.openedx.core.system.notifier
+
+import org.openedx.core.system.notifier.app.AppEvent
+
+sealed class PushEvent : AppEvent {
+    data object RefreshPushEvent : PushEvent()
```

Comment:
We need a more descriptive name for this event, such as `RefreshBadgeCount`, to better reflect its functionality.

Reply:
renamed with the suggested one.